packages/packages/commonwell-sdk/src/client/commonwell.ts

```diff
@@ -77,6 +85,255 @@ export class CommonWell {
+   *
+   * @param meta         Metadata about the request.
+   * @param certificate  The certificate to add to the org
+   * @param id           The org to get a certificate from
+   * @param thumbprint   The thumbprint from the certificate
+   * @param purpose      The purpose from the certificate
+   * @returns
+   */
+  async getCertificateFromOrg(
+    meta: RequestMetadata,
+    id: string,
+    thumbprint?: string,
+    purpose?: string
+  ): Promise<CertificateResp> {
+    const headers = await this.buildQueryHeaders(meta);
+    const resp = await this.api.get(
+      `${CommonWell.MEMBER_ENDPOINT}/${this.oid}/org/${id}/certificate`,
+      {
+        headers,
+        params: { thumbprint, purpose },
+      }
+    );
+    return certificateRespSchema.parse(resp.data);
+  }
+
+  /**
+   * Get certificate from org (by thumbprint).
+   * See: https://commonwellalliance.sharepoint.com/sites/ServiceAdopter/SitePages/Organization-Management-API---Overview-and-Summary.aspx#get-certificates-by-thumbprint
+   *
+   * @param meta         Metadata about the request.
+   * @param certificate  The certificate to add to the org
+   * @param id           The org to get a certificate from
+   * @param thumbprint   The thumbprint from the certificate
+   * @param purpose      The purpose from the certificate
+   * @returns
+   */
+  async getCertificateFromOrgByThumbprint(
+    meta: RequestMetadata,
+    id: string,
+    thumbprint: string,
+    purpose?: string
+  ): Promise<CertificateResp> {
+    const headers = await this.buildQueryHeaders(meta);
+    const resp = await this.api.get(
+      `${CommonWell.MEMBER_ENDPOINT}/${this.oid}/org/${id}/certificate/${thumbprint}`,
+      {
+        headers,
+        params: { purpose },
+      }
+    );
+    return certificateRespSchema.parse(resp.data);
+  }
+
+  /**
+   * Get certificate from org (by thumbprint & purpose).
+   * See: https://commonwellalliance.sharepoint.com/sites/ServiceAdopter/SitePages/Organization-Management-API---Overview-and-Summary.aspx#get-certificates-by-thumbprint-and-purpose
+   *
+   * @param meta         Metadata about the request.
+   * @param certificate  The certificate to add to the org
+   * @param id           The org to get a certificate from
+   * @param thumbprint   The thumbprint from the certificate
+   * @param purpose      The purpose from the certificate
+   * @returns
+   */
+  async getCertificateFromOrgByThumbprintAndPurpose(
+    meta: RequestMetadata,
+    id: string,
+    thumbprint: string,
+    purpose: string
+  ): Promise<CertificateResp> {
+    const headers = await this.buildQueryHeaders(meta);
+    const resp = await this.api.get(
+      `${CommonWell.MEMBER_ENDPOINT}/${this.oid}/org/${id}/certificate/${thumbprint}/purpose/${purpose}`,
+      {
+        headers,
+      }
+    );
+    return certificateRespSchema.parse(resp.data);
+  }
```

Comment:
One of the strangest API designs I've ever seen - why do these 2 endpoints exist if the first one with the query params already does? 🤷 

In any case, as per the previous comment, these can return multiple certs so `getCertificates` in both names would be better

Reply:
Yea i didnt understand this but i made them just in case